src/System.Buffers/src/project.json

```diff
@@ -1,11 +1,16 @@
 {
+  "compilationOptions": {
+    "allowUnsafe": true
+  },
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.1-beta-23409",
     "System.Reflection.Primitives": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.Extensions": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+    "System.Slices": "0.1.0-d103015-1",
```

Comment:
Can we keep project to project references for dlls in corefxlab? We often need to change more than one dll and having to publish to nuget makes the cycle very long.